website/src/components/Changelog/GUI.tsx

```diff
@@ -8,19 +8,20 @@ export default function GUI({ os }: { os: OS }) {
   return (
     <Entries downloadLinks={downloadLinks(os)} title={title(os)}>
       {/* When you cut a release, remove any solved issues from the "known issues" lists over in `client-apps`. This must not be done when the issue's PR merges. */}
-      <Unreleased>
+      <Unreleased></Unreleased>
+      <Entry version="1.4.7" date={new Date("2025-02-26")}>
         {os === OS.Linux && (
           <ChangeItem pull="8219">
             Configures the IPC service to log to journald.
           </ChangeItem>
         )}
         {os === OS.Windows && (
           <ChangeItem pull="8268">
-            Fixes a dead-lock that could occur during shutdown of the TUN
-            device if there were still packets queued for sending.
+            Fixes a dead-lock that could occur during shutdown of the TUN device
```

Comment:
Oh, the changelog update I made disappeared in the rebase I think. I actually liked the way you had worded this (modulo the error code).

Reply:
K will fix it.